### ISSUE TITLE
Make all text on green cards visible on canvas

### ISSUE THREAD

**Issue by askbulle** [equinor/MAD-VSM-WEB#152]
Several users map processes with a very long lists of suppliers, input, output and customers. Sometimes this is due to very complex process, sometimes just very bad and/or unclear process when these lists are long. In both cases seeing the entire list is valuable. 

As of now, only the top of these lists are visible on the canvas.

Suggested solutions
a. automatically expanding green cards vertically to show all text
b. make it possible to add more green cards vertically

**Comment by knudsvik** [comment]
Agreed to keep size as is, but have an indicator of more text available and hover functionality to see all text.

**Comment by Linedueo** [comment]
https://user-images.githubusercontent.com/74361011/150541845-512e7d41-1f6f-4275-a224-b4b207e51dd3.mov

@knudsvik @SjoenH

**Comment by Linedueo** [comment]
Tror ikke vi skal legge på et ikon som indikerer mer tekst. Det tar plass i kortet og er heller ikke noe du kan trykke på...så jeg tror vi bare beholder (...)

**Comment by SjoenH** [comment]
So, we decided on doing nothing?
Hover text as shown in the video above should be covered by:  #253

**Comment by knudsvik** [comment]
> So, we decided on doing nothing? Hover text as shown in the video above should be covered by: #253

We did agree on hover, but as you say, it should be covered by #253 it seems.

**Comment by NureHassen1** [comment]
Jg

**Comment by KaterynaBasik** [comment]
It's patially solved in #253

**Comment by askbulle** [comment]
Due to hover functionality #253 this issue is less  important. Moving back to Flyt Roadmap
